samples/DemoWebApplication/Program.cs

```diff
@@ -15,10 +15,11 @@ public class Program
         public static void Main(string[] args)
         {
             CreateWebHostBuilder(args).Build().Run();
-        }
-
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
+        }
+
+		public static IWebHostBuilder CreateWebHostBuilder(string[] args)
```

Comment:
Expression method on this position is used in every sample from MS. So I would keep it.